src/d3a/events/__init__.py

```diff
@@ -57,7 +57,7 @@ def event_listener(self, event_type: Union[AreaEvent, MarketEvent], **kwargs):
         self.log.trace("Dispatching event %s", event_type.name)
         self._event_mapping(event_type)(**kwargs)
 
-    def event_tick(self, *, area):
+    def event_tick(self):
```

Comment:
I like it a lot now that the strategy events are not dependent on the area object, thanks!